pkg/cdi/default-cache_test.go

```diff
@@ -115,7 +115,7 @@ devices:
 					filepath.Join(dir, "run"),
 				),
 			}
-			Configure(opts...)
+			_ = Configure(opts...) // always nil now, should we update and break API ?
```

Comment:
I think we said that we could foresee a future where the configure call MAY return an error.